workflow-templates/checkov.yml

```
@@ -0,0 +1,12 @@
+name: Checkov GHAS Submission
+
+on:
+  push:
+    branches: [ $default-branch ]
+
+permissions:
+  contents: write
+
+jobs:
+  dependency-submission:
+    uses: govuk-one-login/.github/.github/workflows/checkov.yml@main
```

Outstanding ticket next needs to be to change this to a release process e.g. `@v1` instead of `@main` before it gets widely used.

---

```suggestion
    uses: govuk-one-login/.github/.github/workflows/checkov.yml@e3caa6e3d5fc7b3794bec3c00f9d8db1e7446888 # v0
```

Does this satisfy the requirement, if we tag this merge commit as version 0?

---

@cjfryer if you're happy with that approach, dismiss this review and merge and call me and we'll do a follow up merge.